tests/foreman/ui/test_host.py

```diff
@@ -3127,3 +3127,87 @@ def _cleanup():
             assert host.compute_resource is None, (
                 f"Compute resource ID for {vm_name} is not None after disassociation"
             )
+
+
+def test_positive_change_org_loc_of_multi_hosts(
```

Comment:
```suggestion
def test_positive_change_hosts_org_loc(
```
I would suggest shortening the name of the test like this, if you agree, so it corresponds to the naming scheme in other tests related to this feature